tests/components/switcher_kis/test_switch.py

```diff
@@ -97,18 +98,19 @@ async def test_switch_control_fail(
 
     # Test exception during turn on
     with patch(
-        "homeassistant.components.switcher_kis.switch.SwitcherApi.control_device",
+        "homeassistant.components.switcher_kis.entity.SwitcherApi.control_device",
         side_effect=RuntimeError("fake error"),
     ) as mock_control_device:
-        await hass.services.async_call(
-            SWITCH_DOMAIN, SERVICE_TURN_ON, {ATTR_ENTITY_ID: entity_id}, blocking=True
-        )
+        with pytest.raises(HomeAssistantError):
+            await hass.services.async_call(
+                SWITCH_DOMAIN,
+                SERVICE_TURN_ON,
+                {ATTR_ENTITY_ID: entity_id},
+                blocking=True,
+            )
 
         assert mock_api.call_count == 2
         mock_control_device.assert_called_once_with(Command.ON)
-        assert (
-            f"Call api for {device.name} failed, api: 'control_device'" in caplog.text
-        )
```

Comment:
Please remove the unused `caplog`

Reply:
It is done.

Reply:
Remove the `caplog` fixture as it is not used https://github.com/home-assistant/core/blob/1aad3f8a0f9e070e129f9cdc2a4582c07a5cb00b/tests/components/switcher_kis/test_switch.py#L82

Reply:
It is done.